packages/playground/src/pages/GiftCards/GiftCards.js

```diff
@@ -76,9 +76,8 @@ import '../../style.scss';
         .create('giftcard', {
             type: 'giftcard',
             brand: 'svs',
-            onOrderCreated: async (resolve, reject, data) => {
+            onOrderCreated: async (data) => {
```

Comment:
Is there is a need for the `async` prefix in the function and also for the `afterGiftCard` to be async? I believe that there is no need for that here. The code is all synchronous

Reply:
Kinda just left it for testing purposes. I agree this makes it more clear.